DEEPRAJKUSHWAHA.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "DEEPRAJ KUSHWAHA",
+  "type": "personal",
+  "bio": "I am a android and full stack developer"
```

There is a missing comma which is leading json check fail. 

```suggestion
  "bio": "I am a android and full stack developer",
```